backend/inspirehep/editor/views.py

```diff
@@ -250,3 +251,14 @@ def upload():
     )
     file_url = current_s3_instance.get_s3_url(key, bucket)
     return jsonify({"path": file_url}), 200
+
+
+@blueprint.route("/authorlist/text", methods=["POST"])
+@login_required_with_roles([Roles.cataloger.value])
+def authorlist_text():
+    """Run authorlist on a piece of text."""
+    try:
+        parsed_authors = authorlist(request.json["text"])
+        return jsonify(parsed_authors)
+    except Exception as err:
+        return jsonify(status=500, message=" / ".join(err.args)), 500
```

Comment:
wouldn't this be better as a `400`, as it's not really something that blew up, but rather incorrect input that the script doesn't understand. Or does the record editor expect a `500` here?

Reply:
Yes, it's required by editor but it seems that it can be changed easily:
https://github.com/inspirehep/inspirehep/blob/f9c41f9578cd0c6aea6f4e2dc78236c8b5c34ea9/record-editor/src/app/shared/components/extract-actions/author-extract-actions.component.ts#L100-L109